### ISSUE TITLE
Invalid position given by 'W030'

### ISSUE THREAD

**Issue by welectronic** [AtomLinter/linter-jshint#487]
JSHint returned a point that did not exist in the document being edited.
Rule: `W030`
Requested point: 372:41

I'm using jquery.validate.js lib
```js
		elements: function() {
			var validator = this,
				rulesCache = {};

			// select all valid inputs inside the form (no submit or reset buttons)
			// workaround $Query([]).add until http://dev.jquery.com/ticket/2114 is solved
			return $([]).add(this.currentForm.elements)
			.filter(":input")
			.not(":submit, :reset, :image, [disabled]")
			.not( this.settings.ignore )
			.filter(function() {
				!this.name && validator.settings.debug && window.console && console.error( "%o has no name assigned", this);

				// select only the first element for each name, and only those with rules specified
				if ( this.name in rulesCache || !validator.objectLength($(this).rules()) )
					return false;

				rulesCache[this.name] = true;
				return true;
			});
		}
```

Debug information:
```
Atom version: 1.19.2
linter-jshint version: v3.1.5
JSHint version: jshint v2.9.5
Hours since last Atom restart: 0.4
Platform: win32
Current file's scopes: [
  "source.js",
  "punctuation.terminator.statement.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

**Comment by Arcanemagus** [comment]
Please see the following issue for more information.

**Comment by Arcanemagus** [comment]
Duplicate of #421
